test/unit/forge/routes/api/user_spec.js

```diff
@@ -101,7 +105,36 @@ describe('User API', async function () {
             result.should.have.property('id', TestObjects.alice.hashid)
             result.should.have.property('username', TestObjects.alice.username)
             result.should.have.property('email', TestObjects.alice.email)
+            result.should.not.have.property('sso_enabled')
+        })
+        const ssoUserInfoTest = it('return user info for logged in user - sso enabled', async function () {
+            // Check !sso_enabled
+            await login('alice', 'aaPassword')
+            const response = await app.inject({
+                method: 'GET',
+                url: '/api/v1/user',
+                cookies: { sid: TestObjects.tokens.alice }
+            })
+            response.statusCode.should.equal(200)
+            const result = response.json()
+            result.should.have.property('id', TestObjects.alice.hashid)
+            result.should.have.property('username', TestObjects.alice.username)
+            result.should.have.property('email', TestObjects.alice.email)
+            result.should.have.property('sso_enabled', false)
```

Comment:
This fails because the view only adds the SSO field if it's set to true

Reply:
Odd - tests passes locally. Will investigate.